test/buy-collateral-test.ts

```diff
@@ -409,7 +409,7 @@ describe('buyCollateral', function () {
   });
 
   describe('reentrancy', function() {
-    it('is not broken by reentrancy supply ', async () => {
+    it('is blocked during reentrant supply', async () => {
```

Comment:
Can you assert that the call reverts with `ReentrantCallBlocked`?

Reply:
Added another small test case in the next to test on revert.